randovania/data/json_data/prime3.txt

```diff
@@ -2209,9 +2667,13 @@ Asset id: 4133901881028310574
 > Dock to Main Docking Access; Heals? False
   * Normal Door to Main Docking Access/Dock to Main Docking Bay
   > Pickup (Missile Expansion)
-      Plasma Beam
+      Plasma Beam and After Ghor
```

Comment:
Should be (plasma or Nova) and ghor. Probably with adding a template for this called "Melt Objects"?